### DISCUSSION TITLE
[DOC] Hugepage-2Mi recommended

### DISCUSSION THREAD

**Discussion by ElectroshockGuy** [longhorn/longhorn#8592] [General]
## What's the document you plan to update? Why? Please describe

Hello, teams.
Currently, it has been found that the size setting of hugepage-2mi affects the number of v2 PVC creations. Can you provide a recommended list or calculation formula in the document so that we can understand our own needs and set an appropriate size for hugepage-2mi before using it?

**Comment by derekbit** [comment]
cc @shuo-wu @DamiaSan as well
